src/themes/palette.ts

```diff
@@ -1,67 +1,114 @@
-  alertSecondaryText: '#41464b',
-  alertSuccessText: '#0f5132',
-  alertWarningText: '#664d03',
-  alertErrorText: '#842029',
-  alertInfoText: '#055160',
-  alertPrimaryBackground: '#cfe2ff',
-  alertSecondaryBackground: '#e2e3e5',
-  alertSuccessBackground: '#d1e7dd',
-  alertWarningBackground: '#fff3cd',
-  alertErrorBackground: '#f8d7da',
-  alertInfoBackground: '#cff4fc',
-  alertPrimaryBorder: '#b6d4fe',
-  alertSecondaryBorder: '#d3d6d8',
-  alertSuccessBorder: '#badbcc',
-  alertWarningBorder: '#ffecb5',
-  alertErrorBorder: '#f5c2c7',
-  alertInfoBorder: '#b6effb',
+//   // BACKGROUNDS
+//   backgroundPrimary: '#422ed1',
+//   backgroundSecondary: '#rgba(142, 144, 166, 1)',
+//   backgroundSuccess: '#198754',
+//   backgroundError: '#dc3545',
+//   backgroundWarning: '#ffc107',
+//   backgroundLight: '#f8f9fa',
+//   backgroundDark: '#212529',
+//   backgroundTransparent: '#transparent',
+//   backgroundWhite: '#fff',
+//   backgroundAccent: '#F6F7FB',
 
-  // BACKGROUNDS
-  backgroundPrimary: '#422ed1',
-  backgroundSecondary: '#rgba(142, 144, 166, 1)',
-  backgroundSuccess: '#198754',
-  backgroundError: '#dc3545',
-  backgroundWarning: '#ffc107',
-  backgroundLight: '#f8f9fa',
-  backgroundDark: '#212529',
-  backgroundTransparent: '#transparent',
-  backgroundWhite: '#fff',
-  backgroundAccent: '#F6F7FB',
+//   // BUTTONS -background
 
-  // BUTTONS -background
+//   buttonPrimary: '#422ed1',
+//   buttonPrimaryDisabled: 'rgba(66, 46, 209, 0.45)',
+//   buttonSecondary: 'rgba(142, 144, 166, 1)',
+//   buttonSecondaryBorder: 'rgba(142, 144, 166, 1)',
+//   buttonSecondaryHover: '#fff',
+//   buttonSecondaryDisabled: 'rgba(66, 46, 209, 0.45)',
+//   buttonGhost: '#fff',
+//   buttonGhostHover: '#422ed1',
+//   buttonGhostText: '#8E90A6',
+//   buttonGhostBorder: '#8E90A6',
+//   ghostHoverText: '#422ed1',
 
-  buttonPrimary: '#422ed1',
-  buttonPrimaryDisabled: 'rgba(66, 46, 209, 0.45)',
-  buttonSecondary: 'rgba(142, 144, 166, 1)',
-  buttonSecondaryBorder: 'rgba(142, 144, 166, 1)',
-  buttonSecondaryHover: '#fff',
-  buttonSecondaryDisabled: 'rgba(66, 46, 209, 0.45)',
-  buttonGhost: '#fff',
-  buttonGhostHover: '#422ed1',
-  buttonGhostText: '#8E90A6',
-  buttonGhostBorder: '#8E90A6',
-  ghostHoverText: '#422ed1',
+//   buttonSuccess: '#0B7247',
+//   buttonSuccessActive: '#276738',
+//   buttonSuccessHover: '#367b48',
 
-  buttonSuccess: '#0B7247',
-  buttonSuccessActive: '#276738',
-  buttonSuccessHover: '#367b48',
+//   buttonWarning: '#CAB23F',
+//   buttonWarningActive: '#b49e35',
 
-  buttonWarning: '#CAB23F',
-  buttonWarningActive: '#b49e35',
+//   buttonWarningHover: '#cab23f',
+//   buttonError: '#DB2A30',
+//   buttonErrorActive: '#DB2A30',
+//   buttonErrorHover: '#b54248',
+// };
+// export type Palette = typeof palette;
 
-  buttonWarningHover: '#cab23f',
-  buttonError: '#DB2A30',
-  buttonErrorActive: '#DB2A30',
-  buttonErrorHover: '#b54248',
+export const palette = {
+  green: {
+    greenDefault: '#2ba745',
```

Comment:
Its not necessary to rewrite the color name since these properties are already contained inside that color. Example:

green: {
  main: '#0f0',
  light: '#123',
  lighter: '#123',
  dark: '#123',
  darker: '#123',
  translucent: '#123',
  contrastText: '#123',
}

Reply:
Please create an interface as well to maintain consistency.